google/cloud/spanner/CMakeLists.txt

```diff
@@ -300,7 +299,6 @@ function (spanner_client_define_tests)
 
     set(spanner_client_unit_tests
         # cmake-format: sortable
-        connection_options_test.cc
```

Comment:
Shouldn't we keep this test?

Reply:
Yes, you're right. Good catch. I accidentally deleted that one. PR coming up.